testhelper/certificatetest.go

```diff
@@ -111,6 +131,21 @@ func getRSACertTuple(cn string, issuer *RSACertTuple) RSACertTuple {
 	return GetRSACertTupleWithPK(pk, cn, issuer)
 }
 
+func getRevokableRSACertTuple(cn string, issuer *RSACertTuple) RSACertTuple {
+	template := getCertTemplate(issuer == nil, true, cn)
+	template.OCSPServer = []string{"http://localhost:8080/ocsp"}
+	return getRSACertTupleWithTemplate(template, issuer.PrivateKey, issuer)
+}
+
+func getRevokableRSAChainCertTuple(cn string, previous *RSACertTuple, index int) RSACertTuple {
+	template := getCertTemplate(previous == nil, true, cn)
+	template.BasicConstraintsValid = true
+	template.IsCA = true
+	template.KeyUsage = 0
+	template.OCSPServer = []string{fmt.Sprintf("http://localhost:8080/chain_ocsp/%d", index)}
```

Comment:
```suggestion
	template.OCSPServer = []string{fmt.Sprintf("http://someurl.com/someocspendpoint/%d", index)}
```

Reply:
I agree that that would be best to convey that we aren't actually using localhost. However, if this is used elsewhere, it wouldn't be great to make an actual request to a random site. What do you think of `http://example.com/chain_ocsp/%d`?